lib/datadog/tracing/contrib/aws/instrumentation.rb

```diff
@@ -37,8 +37,10 @@ def annotate!(span, context)
             span.set_tag(Tracing::Metadata::Ext::TAG_OPERATION, Ext::TAG_OPERATION_COMMAND)
 
             # Tag as an external peer service
-            span.set_tag(Tracing::Metadata::Ext::TAG_PEER_SERVICE, span.service)
-            span.set_tag(Tracing::Metadata::Ext::TAG_PEER_HOSTNAME, context.safely(:host))
+            if Datadog.configuration.tracing.span_attribute_schema == 'v0'
```

Comment:
The string `'v0'` is being repeated many times, I recommend putting it into a constant.

Reply:
The conditional for checking schema is also expected to be repeated by multiple integrations, I recommend the extract this.